articles/app-service/quickstart-php.md

```diff
@@ -40,20 +41,39 @@ To complete this quickstart, you need:
     cd php-docs-hello-world
     ```
 
-1. To run the application locally, use the `php` command to launch the built-in PHP web server.
+2. To run the application locally, use the `php` command to launch the built-in PHP web server.
```

Comment:
Use `1.` for all will automatically order the numbered list.

Reply:
[Resolved](https://github.com/MicrosoftDocs/azure-docs/pull/93186/commits/544627bc723d0e9ab18e240af1563d9af05cc3e5)